docker-compose-developer.yml

```diff
@@ -276,7 +280,7 @@ services:
     container_name: postgres_to_create_grafana_db
     labels: *oncall-labels
     image: postgres:14.4
-    command: >
```

Comment:
`>-` removes trailing newlines.. I suppose this causes an issue for `mysql`? 

https://stackoverflow.com/a/21699210